pkg/patterns/declarative/annotations.go

```diff
@@ -0,0 +1,22 @@
+package declarative
+
+import (
+	"context"
+
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/kubebuilder-declarative-pattern/pkg/patterns/declarative/pkg/manifest"
+)
+
+// AddAnnotations returns an ObjectTransform that adds annotations to all the objects
+func AddAnnotations(annotations map[string]string) ObjectTransform {
+	return func(ctx context.Context, o DeclarativeObject, manifest *manifest.Objects) error {
+		log := log.Log
+		// TODO: Add to selectors and labels in templates?
```

Comment:
I guess this was copy-pasted from AddLabels?  I don't think it applies to annotations.

Reply:
Yeah, it was a copy-paste.
```suggestion
```